# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/qt-uclibc.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- qt-x11-free-3.3.4/src/tools/qlocale.cpp.orig	2005-02-01 09:45:51.000000000 +0000
+++ qt-x11-free-3.3.4/src/tools/qlocale.cpp	2005-02-01 09:46:26.000000000 +0000
@@ -55,6 +55,8 @@
 #   undef INFINITY
 #endif
 
+#undef Q_OS_LINUX
+
 #ifdef Q_OS_LINUX
 #    include <fenv.h>
 #endif
